_components/customisation.md

```diff
@@ -0,0 +1,199 @@
+
+This document aims to help you decide if you should customise a component for your project, show the ways a component may be customised, and demonstrate why other approaches should be avoided.
+
+<abbr title="Too long; didn't read">
+<strong>
+TL;DR:
+</strong>
+</abbr>
+- Double check customisation is appropriate. Reusing an existing pattern or updating a component to support and share your usecase is preferable.
+- Specify a brand which suits your type of project.
+- Customise component's using their Sass API.
+- Avoid CSS overrides, they are unsupported and unreliable.
+
+## Check Customisation Is Appropriate
+
+The first step before customising an Origami component is to ask:
+1. Is there an existing style which could work already? After searching the [Origami Registry](https://registry.origami.ft.com/components) it's worth double checking in the <a href="https://financialtimes.slack.com/messages/{{site.data.contact.slack}}">#{{site.data.contact.slack}}</a> Slack channel, to reduce duplicated effort and to help maintain a consistent user interface across FT Group projects.
+2. Would your customisation be useful for other teams/projects? Maybe a component should be updated or added to for your usecase, so other teams benefit from your work. A new component might be more appropriate if your design diverges significantly.
+
+A good place to start is to ask for feedback in the <a href="https://financialtimes.slack.com/messages/{{site.data.contact.slack}}">#{{site.data.contact.slack}}</a> Slack channel. The core Origami team, along with the design team, can help identify similar existing styles or help make any relevant component updates to support your usecase without customisation.
+
+## Brands
+
+Most Origami components are branded to provide a distinct appearance within different contexts. A brand may be thought of as a kind of theme, but applies to all components within a project.
+
+### Supported brands
+
+Origami maintained brands include:
+- master: FT branding for public ft.com sites and affiliates.
+- internal: Style suitable for internal products, tools, and documentation.
+- whitelabel: Base, structural styles only to build on and customise.
+
+Taking `o-table` as an example, the "master" brand version offers a "row stripes" feature, which uses "master" brand colours such as "paper" and "wheat":
+{% include demo.html component="o-table" demo="row-stripes" brand="master" height="242px" %}
+
+The "internal" brand version also supports "row stripes" but uses a different colour palette:
+{% include demo.html component="o-table" demo="row-stripes" brand="internal" height="242px" %}
+
+The "whitelabel" brand does not support "row stripes" and therefore outputs no styles to support that feature:
+{% include demo.html component="o-table" demo="basic" brand="whitelabel" height="256px" %}
```

Comment:
This demo currently gets clipped on the right.

![image](https://user-images.githubusercontent.com/1569131/111618893-d15cb200-87dc-11eb-8046-04705afe6ab6.png)

Reply:
The demo does in the registry too. I can live with that. We don't have a responsive demo with stripes, which I think is helpful to show brands.